src/kinetics/include/antioch/kinetics_enum.h

```diff
@@ -68,6 +68,16 @@ namespace Antioch
       return 1.0; // this HAS to stay this way because it is hard-coded for performances (see eq. above)
     }
 
+    enum Parameters{ A = 0,
+                     B,
+                     E,
+                     D,
+                     T_REF,
+                     R_SCALE,
+                     SIGMA,
+                     LAMBDA
+                   };
+
```

Comment:
I'm not yet at this part, the string will be given at the ```ReactionSet``` level, which will perform both the reaction scanning then the string to enum conversion (using a free method of _string_utils.h_, to be implemented).  I'm leaning towards a switch statement and not building a map, as all the keywords will all be at the same easily `foundable' place.